README.md

```diff
@@ -61,9 +62,10 @@ Your web application registration should include the following information:
 - Provide a descriptive Name for your web application, for example, `My Test ASP.NET Web Application`. You can identify this application by its Name within the Azure portal.
 - Set the Reply URL to `https://localhost:44316/` This is the port number that this ASP.NET Web Application sample is configured to run on.
 - Create your application.
-- Once the application is created, from the menu select **Authentication**. In the **Implict grant** section, select **Access tokens**.
-- Next, create a Web App client secret. In the Azure portal go to your **Azure AD B2C** instance. From the menu select **App registration**. Select the registration for your Web Application. From the menu select **Certificates & secrets** and click **New client secret**. Note: You will only see the secret once. Make sure you copy it.
+- Once the application is created, from the menu select **Authentication**. In the **Implicit grant** section, select **Access tokens**.
```

Comment:
Do we really need implicit grant access tokens for a web app? that doesn't sound right to me?